### PR TITLE
Work results card

### LINKED ISSUE #31: Add error handling for when user leaves a language/method field blank

### LINKED ISSUE #32: Add function to clear selections upon "reset" button click and "translation" button click

### PR DESCRIPTION
**Pull request by VeeAndrade**
### Sidebar Checklist
​
- [x] Ran test suite
- [x] Request reviewers
- [x] Assign yourself and other contributors
- [x] Link to project
- [ ] Added to Readme
​
### Issues Resolved
Resolves #31 resolves #32  resolves#35 
​
### Problem Addressed
​UI/UX needed improvement for a presentable application
### Solution Implemented
​Implemented conditional rendering for welcome card and for error handling, also implemented a refresh on the reset button to allow the user to start over
### Screenshots (optional)
​
### Other Notes

### CODE REVIEW

File: src/components/IntroCard/IntroCard.jsx [hunk 1]
```diff
@@ -10,7 +10,10 @@ export const IntroCard = () => {
       <section className='intro-message'>
         <p className='intro-para'>Rosetta was designed to help those who are
         interested in learning a new coding language and want to know the basics.
-        Just follow these 3 steps to get started:
+        Rosetta has a webscraping tool that surfs the web for you and fins your best
```
Comment (mintona) [inline src/components/IntroCard/IntroCard.jsx:13]: Quick typo fix - fins --> finds

File: src/components/ResultCard/ResultCard.jsx [hunk 2]
```diff
@@ -24,8 +24,19 @@ export const ResultCard = ({rating, method}) => {
...
         </div>
+      <section className='user-vote-section'>
+        <button className='vote-btn'>
+          This wasn't helpful
+          <span className='down-vote'></span>
+        </button>
+        <button className='vote-btn'>
+          This was helpful
+          <span className='up-vote'></span>
+        </button>
```
Comment (mintona) [inline src/components/ResultCard/ResultCard.jsx:38]: Yes buttons!

File: src/components/SourceBlock/SourceBlock.jsx [hunk 3]
```diff
@@ -13,8 +13,13 @@ return (
...
         </select>
       </section>
+      { error ?
+      <section className='input-error'>
+        <p className='error-message'>**You must fill out all inputs**</p>
+      </section> : null
+      }
       <section className='action-buttons-sect'>
-        <button className='reset-button'>Reset</button>
+        <button className='reset-button' onClick={() => clearDropBoxes()}>Reset</button>
```
Comment (mintona) [inline src/components/SourceBlock/SourceBlock.jsx:22]: Nice job on the reset functions.

File: src/components/IntroCard/IntroCard.jsx (hunk 1 above)
Comment (jobannon) [inline src/components/IntroCard/IntroCard.jsx:13]: finds * sp error

File: src/styles/Loading.scss [hunk 4]
```diff
@@ -3,3 +3,101 @@ $primary-color: #2B2C28;
 $darker-blue: #339989;
 $pale-blue: #7DE2D1;
 $snow: #FFFAFB;
+
+.cs-loader {
+  position: absolute;
```
Comment (mintona) [inline src/styles/Loading.scss:8]: Love the new loader!

File: src/components/Loading/Loading.jsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+          <label>●</label>
+          <label>●</label>
+          <label>●</label>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export default Loading
```
Comment (DanielEFrampton) [inline src/components/Loading/Loading.jsx:21]: Should we add some kind of attribution to the codepen author?

**Review by mintona — APPROVED** [review]
Thanks, Vee. Making great progress here!

**Review by DanielEFrampton — APPROVED** [review]
Looking great!! Thanks for squashin' them bugs

**Review by jobannon — APPROVED** [review]

**Pull request merged**